backend/src/main/java/com/redhat/cloud/notifications/routers/EventService.java

```diff
@@ -50,7 +50,7 @@
 
     @GET
     @Produces(APPLICATION_JSON)
-    @RolesAllowed(RBAC_READ_NOTIFICATIONS)
+    @RolesAllowed(RBAC_READ_EVENTS_NOTIFICATIONS)
```

Comment:
Could you add a test that tries to access this API with the `RBAC_READ_NOTIFICATIONS` permission?

Reply:
added.